analysis/src/Event.cpp

```diff
@@ -94,7 +122,7 @@ PhysObjects Event::GetGoodPhotons()
     if(photon->GetEt() < config.params("photonMinEt")) continue;
     
     // Check swiss cross
-    double E4 = photon->GetEnergyCrystalTop() +
```

Comment:
Why did you comment out swiss-cross check?

Reply:
Uncommitted it and updated the number to 0.0 in config.